web/packages/build/package.json

```diff
@@ -98,10 +86,7 @@
     "vite": "^5.3.1",
     "vite-plugin-wasm": "^3.3.0",
     "vite-tsconfig-paths": "^4.3.2",
-    "webpack": "^5.76.2",
-    "webpack-bundle-analyzer": "^4.10.2",
-    "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.9.3"
+    "webpack": "^5.76.2"
```

Comment:
Are we able to get rid of webpack too? `@storybook/builder-webpack5` already has it in its dependencies. We import it through TS comments in the config files, but at this point I think we can get rid of that.